includes/core/server/class-request.php

```diff
@@ -226,6 +229,33 @@ final public function send( $hook, ...$args ) {
 		);
 	}
 
+	/**
+	 * This is mimic of send method, but where API execption is handled.
+	 * The reason behind this is that sometimes API request can fail for valid reasons and instead of handling this exception on every request, you could use this function.
+	 *
+	 * @param string $hook         The filter to use.
+	 * @param mixed  ...$args      Other parameters for the hook.
+	 * @return mixed               Either the response array, or the correct object.
+	 *
+	 * @throws Extend_Request_Exception
+	 * @throws Immutable_Parameter_Exception
+	 * @throws Invalid_Request_Parameter_Exception
+	 */
+	final public function handle_rest_request( $hook, ...$args ) {
+		try {
+			$data = $this->send( $hook, $args );
```

Comment:
Don't we need to expand `$args` here as well?
```suggestion
			$data = $this->send( $hook, ...$args );
```

Reply:
Fixed.